ci/run-integration-tests.ps1

```diff
@@ -0,0 +1,20 @@
+param (
+    [Parameter(Mandatory=$true)]
+    [string]$RepoName
+)
+
+Push-Location $RepoName
+
+try
+{
+    Write-Output "Running integration tests"
+    $env:RESOURCE_KEY = $Options.Keys.TestResourceKey
```

Comment:
Options isn't actually named as a parameter in this script. I didn't spot this in device-detection and pipeline, but could you double check?

Reply:
Note: Actually it's kinda weird that was working just fine, i'll change add $Keys param to make it more obvious
i think $Options is something similar to ...args in js

Reply:
Maybe it's available in the frame above, and is accessible here too. But the change you made is better.